doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -359,7 +361,7 @@ JCL      - e5c64f5 based on jdk-11+21)
 ## Windows
 :ledger:
 
-The following instructions guide you through the process of building a Windows **OpenJDK V11** binary that contains Eclipse OpenJ9. This process can be used to build binaries for Windows 7, 8, and 10.
+The following instructions guide you through the process of building a Windows **OpenJDK V11** binary that contains Eclipse OpenJ9. This process can be used to build binaries for Windows 8 and 10.
```

Comment:
How about we just say Windows instead of listing versions. Already we're missing Windows Server. When we don't need to keep updating it.

Reply:
Fixed in all sets of instructions.